src/bin/tokenrules.js

```diff
@@ -0,0 +1,44 @@
+#!/usr/bin/env node
+
+'use strict';
+
+const program = require('commander');
+
+const ChainConfig = require('../config/chain_config');
+const Deployer = require('../deployer.js');
+const logger = require('../logger');
+
+const { version } = require('../../package.json');
+
+program
+  .version(version)
+  .name('TokenRules')
+  .arguments('<config> <organization> <eip20Token>')
+  .description('An executable to deploy TokenRules.')
+  .action(
+    async (config, organization, eip20Token) => {
+      const chainConfig = new ChainConfig(config);
+      const deployer = new Deployer(chainConfig);
+      const tokenRulesAddress = await deployer.deployTokenRules(organization, eip20Token);
+      logger.info(`Deployed TokenRules address: ${tokenRulesAddress}`);
+      chainConfig.update({
+        tokenRulesAddress: tokenRulesAddress,
+      });
+      chainConfig.write(config);
+    },
+  )
+  .on(
+    '--help',
+    () => {
+      console.log('');
+      console.log('Arguments:');
+      console.log('  config       path to a config file');
+      console.log('  organization Auxiliary chain organization address');
+      console.log('  eip20Token   EIP20Token contract address');
```

Comment:
In the future: it should be two spaces after the argument and the description. The description should start lower case.
```js
console.log('  config        path to a config file');
console.log('  organization  auxiliary chain organization address');
console.log('  eip20Token    EIP20Token contract address');
```

Reply:
```
console.log('');
      console.log('Arguments:');
      console.log('  config  path to a config file');
      console.log('  organization  Auxiliary chain organization address');
      console.log('  eip20Token  eip20token contract address');
      console.log('');
      console.log('Examples:');
      console.log('  Deploys JLP TokenRules:');
      console.log('  $ tokenrules.js config.json organization eip20token');
```
Is above fine?

Reply:
No, sorry I wasn't clear. I meant: aligned, with two spaces after the longest path. See my example. It's not important now, let's do it right in the future 👍